components/suggest/src/schema.rs

```diff
@@ -286,6 +292,10 @@ impl ConnectionInitializer for SuggestConnectionInitializer<'_> {
     }
 
     fn upgrade_from(&self, tx: &Transaction<'_>, version: u32) -> open_database::Result<()> {
+        // Custom functions are per connection. `prepare` usually handles
+        // creating them but on upgrade it's not called before this method is.
+        self.create_custom_functions(tx.deref())?;
```

Comment:
Thanks to ["Deref Coercion"](https://doc.rust-lang.org/std/ops/trait.Deref.html#deref-coercion), the compiler will happily take `tx` and implicitly call `tx.deref()` for us.
```suggestion
        self.create_custom_functions(tx)?;
```